File: beginner/__1000__/1035/1035.ts
const { listenerCount } = require("events");
const fs = require("fs")
const input = fs.readFileSync("/dev/stdin", "utf-8")
const [A, B, C, D] = input.split(" ").map(Number)

const condicao = (B > C && D > A && (C + D) > (A + B) && C >= 0 && D >= 0 && A % 2 == 0)

console.log( condicao ? "Valores aceitos" : "Valores nao aceitos" )

File: beginner/__1000__/1035/1035.py
A,B,C,D = [int(el) for el in input().split()]

condicao = (B > C and D > A and (C + D) > (A + B) and C >= 0 and D >= 0 and A % 2 == 0)

if condicao:
    print("Valores aceitos")
else:
    print("Valores nao aceitos")